ahcore/readers.py

```diff
@@ -295,6 +292,74 @@ def read_region(self, location: tuple[int, int], level: int, size: tuple[int, in
 
         return pyvips.Image.new_from_array(stitched_image.transpose(1, 2, 0))
 
+    def _read_feature_region(self, size: tuple[int, int], location: tuple[int, int]) -> pyvips.Image:
+        assert self._num_samples is not None
+
+        image_dataset: h5py.Dataset = self._file["data"]
+
+        assert image_dataset is not None
+
+        x, y = location
+        w, h = size
+
+        if self._stitching_mode != StitchingMode.CROP:
+            raise NotImplementedError("Stitching mode other than CROP is not supported for features.")
+
```

Comment:
You also need to check that the overlap is 0, or at least need to think about what this means for the features, and explain it somewhere.

Reply:
Added check. ATM I don't see how features with overlap should work so we do not allow it.